Gems/Atom/RPI/Code/Source/RPI.Public/WindowContext.cpp

```diff
@@ -13,6 +13,18 @@
 
 #include <Atom/RHI/Factory.h>
 
+#include <AzCore/Console/IConsole.h>
+#include <AzCore/std/containers/unordered_set.h>
+#include <AzCore/std/parallel/mutex.h>
```

Comment:
I don't think these two extra headers are being used.

Reply:
Sorry will be removing it. I originally registered all window handles before I found the ebus broadcast functionality.